i18n/en.json

```diff
@@ -1257,7 +1256,8 @@
   "admin.license.trialCard.purchase": "Purchase",
   "admin.license.trialCard.purchase_license": "Purchase a license",
   "admin.license.trialUpgradeAndRequest.submit": "Upgrade Server And Start trial",
-  "admin.license.upgrade-and-trial-request.accept-terms": "By clicking **Upgrade And Start trial**, I agree to the [Mattermost Software Evaluation Agreement](!https://mattermost.com/software-evaluation-agreement/), [Privacy Policy](!https://mattermost.com/privacy-policy/), and receiving product emails. Also, I agree to the terms of the Mattermost ",
+  "admin.license.upgrade-and-trial-request.accept-terms-final-part": "Also, I agree to the terms of the Mattermost {eeModalTerms}. Upgrading will download the binary and update your team edition.",
+  "admin.license.upgrade-and-trial-request.accept-terms-initial-part": "By clicking **Upgrade And Start trial**, I agree to the [Mattermost Software Evaluation Agreement](!https://mattermost.com/software-evaluation-agreement/), [Privacy Policy](!https://mattermost.com/privacy-policy/), and receiving product emails. ",
```

Comment:
@cwarnermm I kept two different strings since one part of the string can be more easily implemented/maintained using markdown and for implementation restrictions we have to do it this way. However, there is always an alternative to make it into a single string with a little extra cost in terms of code readability and maintenance. Please, let me know what do you think.

Reply:
Thanks, @pablovelezvidal! These two strings read well to target audiences and provide good context to translators. Well done!